### PR TITLE
adding my list

### PR DESCRIPTION
**Pull request by Bri-Eden**

### CODE REVIEW

**Review by ryanmphill — CHANGES_REQUESTED** [review]
Looks like we need to update the PR with the latest changes from main!

File: index.html [hunk 1]
```diff
@@ -25,6 +26,11 @@ <h2>Ryan</h2>
         <li>Earl Grey Tea</li>
     </ul>
 
+<<<<<<< HEAD
```
Comment (ryanmphill) [inline index.html]: When I pull it up in the browser, `<<<<<<< HEAD` and `>>>>>>> main` show up.

**Review by ryanmphill — CHANGES_REQUESTED** [review]
When I pull it up in the browser, '<<<<<<< HEAD' and '>>>>>>>' main show up.

**Review by ryanmphill — CHANGES_REQUESTED** [review]
It looks like Will and Seyun's sections were removed. Can you get the latest main and make sure everyone's info is there?

**Review by stevebrownlee — CHANGES_REQUESTED** [review]
You need to manually add Will and Seyun's lists back into your `index.html` file. They got inadvertantly deleted.

**Review by stevebrownlee — APPROVED** [review]
:shipit:

**Pull request merged**